api/src/main/java/org/openmrs/module/appointments/service/AppointmentsService.java

```diff
@@ -24,7 +24,12 @@ public interface AppointmentsService {
 
     @Transactional
     @Authorized({MANAGE_APPOINTMENTS, MANAGE_APPOINTMENTS})
-    Object sendAppointmentReminderSMS(Appointment appointment);
+    void sendAppointmentReminderSMS(Appointment appointment);
+
+    @Transactional
+    @Authorized({MANAGE_APPOINTMENTS})
```

Comment:
privilege should be different like 
"Send Appointments SMS" combined with "View Appointments" 

This priviledge also should determine whether enabled on the frontend. But for that you may create a different card. Pls create the needed privileges through the config.xml